pkgs/games/itch/default.nix

```diff
@@ -31,7 +31,7 @@ stdenvNoCC.mkDerivation rec {
         owner = "itchio";
         repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21